apps/web/package.json

```diff
@@ -42,6 +42,7 @@
     "@mantine/prism": "4.1.0",
     "@mantine/spotlight": "^5.3.2",
     "@novu/notification-center": "^0.8.0",
+    "@novu/notification-center-react": "^0.0.1",
```

Comment:
Should we keep the versioning aligned to the rest of the Novu packages?

Reply:
yeah, that is the question I had in the notes, what should we do with the previous package and the new one and the versioning. wdyt @novuhq/novu-devs ?

Reply:
I think that the version should be aligned with the rest of the Novu packages, regarding the name we need to decide if we want to make a breaking change or not because '@novu/notification-center-react' sounds great but we may come up with another naming without breaking the packages names.